package.json

```diff
@@ -11,26 +11,27 @@
     "dist"
   ],
   "dependencies": {
-    "chalk": "^2.2.0",
-    "globby": "^6.1.0",
-    "ignore": "^3.3.5",
-    "tslint": "^5.7.0",
-    "yargs": "^9.0.1"
+    "chalk": "^2.3.2",
+    "globby": "^8.0.1",
+    "ignore": "^3.3.7",
+    "require-relative": "^0.8.7",
+    "tslint": "^5.9.1",
+    "yargs": "^11.0.0"
   },
   "peerDependencies": {
-    "prettier": "^1.7.4",
-    "typescript": "^2.5.3"
+    "prettier": "^1.12.0",
+    "typescript": "^2.8.1"
```

Comment:
Is there a reason you increased the `peerDependencies`? I don't think we need any of the newer APIs?

Reply:
no is not required but its good to have last versions

Reply:
This is `peerDependencies`, which forces anyone who uses `prettier-tslint` to upgrade to latest `typescript` and `prettier`, which we don't want to do.